graphql/src/query/ext.rs

```diff
@@ -58,6 +64,19 @@ impl FieldExt for q::Field {
             )
         }
 
+        fn lookup<'a>(
```

Comment:
This looks generally useful, should it go under `FieldExt`?

Reply:
Good point - I pulled it into `ValueExt::lookup`. `Field` got only dragged into it so we would have a position for the error message.

Reply:
Even nicer!